idurar/idurar-erp-crm
# 🇬🇷 Added Greek language #675

## Description

### **Added Greek Language #675 .**

## Related Issues

**Solved #675** 

## Steps to Test
1. You can see the change by just clicking on the dropdown menu for language by selecting to greek.
2. Language will be changed to greek for whole website , working after login also.

## Screenshots (if applicable)
**Login page Screenshot**
![image](https://github.com/idurar/idurar-erp-crm/assets/67959381/a493108c-db1d-4a54-b6b2-a1e59e686ca3)
**After Login Screenshot**
![image](https://github.com/idurar/idurar-erp-crm/assets/67959381/3b25b4d7-2f60-435a-80dd-0a3ba0b10de7)
